src/Feature/Account/client/Integration/Account/sagas.ts

```diff
@@ -152,14 +152,12 @@ export function* addAddress(action: Action<Commerce.Address>) {
     return yield put(actions.UpdateAddressFailure(error.message, error.stack));
   }
 
-  yield put(actions.UpdateAddressSuccess(data));
+  yield put(actions.AddAddressSuccess(data));
```

Comment:
Also in this method:
`UpdateAddressFailure --> AddAddressFailure`
`UpdateAddressRequest--> UpdateAddressRequest`